floris/simulation/turbine.py

```diff
@@ -125,34 +257,21 @@ def power(
     # That said, it adds overhead to convert the floats to 1-D arrays, so I don't
```

Comment:
I think deleting the lines below this means this comment block is no longer relevant and can be deleted